apps/06Checkers.js

```diff
@@ -50,13 +59,35 @@ function Board() {
     }
 
     // Your code here
+    this.createCheckers = function () {
+
+        var whitePositions = [[0, 1], [0, 3], [0, 5], [0, 7],
+[1, 0], [1, 2], [1, 4], [1, 6], [2, 1], [2, 3], [2, 5], [2, 7]];
```

Comment:
Remember to keep an eye on formatting, even though all the  coordinates can't fit on one line, when continued on the next line, it should start at the same column as the coordinates on the line above.